tests/testthat/test-scale.r

```diff
@@ -19,3 +19,24 @@ test_that("NA.value works for discrete", {
   expect_that(dscale(x, pal, "grey50")[1], equals("grey50"))
   expect_that(dscale(x, pal, "grey50")[5], equals("grey50"))
 })
+
+test_that("train_continuous maintains class", {
+  expect_equal(train_continuous(1:5),
+               c(1, 5),
+               info="Normal numeric input works with train_continuous")
```

Comment:
Can you please style your tests according to [style.tidyverse.org](http://style.tidyverse.org)? There should be space around each equal sign. The styler package can help with this.